datasources/new_signal_minute_mv.datasource

```diff
@@ -0,0 +1,15 @@
+# Data Source created from Pipe 'signal_minute'
```

Comment:
According to my previous comment, if the schema of the pipe is compatible with the existing data source there's no need to create a new data source

Reply:
So we just go for the ALTER. Good to pay attention how it goes in the CI as Environment snapshot purpose was to create new VERSION, i.e regression-tests would fail as column is changed for both compared releases